_posts/2022-02-28-war-in-ukraine.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "The War in Ukraine"
+author: [michal.kosmulski]
+tags: [tech, "soft skills", helping]
+---
+The attack on Ukraine has shocked the whole world, but in Poland it is perceived in an especially poignant way.
+Not only is Ukraine our direct neighbor (as are Russia and Belarus), but Poland’s difficult history allows us to
+sympathize with the Ukrainians even more. Many live in Poland, so it’s affecting people we know personally.
+
+While the fighting continues, here’s what [Allegro](https://allegro.tech) is doing, and what you can do, too, to help
+humanitarian efforts.
+
+* If you know any Ukrainians, your coworkers in particular, talk to them and show your support. Even minor gestures are
+  important. Ask directly what kind of help they might need rather than trying to guess.
+* Do not marginalize Russians and Belorussians. People are not the same as their governments, and often do not support
```

Comment:
Shouldn't it be "_Belarussians_"? 🙂 
```suggestion
* Do not marginalize Russians and Belarussians. People are not the same as their governments, and often do not support
```

Reply:
Indeed. That's what I wrote first, then IntelliJ spellcheck suggested this fix, but dictionaries do give the `Belarussians` verison.